apps/ui/src/routes/index.ts

```diff
@@ -39,9 +39,10 @@ router.beforeEach((to, _from, next) => {
   if (isWhiteLabel.value) return next();
 
   const [, space, ...rest] = to.path.split('/');
-  let spaceName = space.replace(`${metadataNetwork}:`, '');
   // skip if network is not metadataNetwork
-  if (spaceName.includes(':')) return next();
+  if (!space.startsWith(`${metadataNetwork}:`)) return next();
+
+  let spaceName = space.replace(`${metadataNetwork}:`, '');
```

Comment:
This was broken, `curtis:0xasdf` was being redirected to `curti0xasdf`.

Reply:
Weird! is there an example space on curtis? 

This is intended initially to redirect https://snapshot.box/#/thanku.eth to https://snapshot.box/#/s:thanku.eth

Reply:
https://snapshot.box/#/curtis:0xf6238F73A87D390CB00b2B380D2B777E13Fe2725
will redirect to
https://snapshot.box/#/s:curti0xf6238F73A87D390CB00b2B380D2B777E13Fe2725

We are replacing `s:` which is intended for `s` network, but it also affects `curtis` (those spaces end with `s:` as well).

Reply:
This kills the original case https://github.com/snapshot-labs/workflow/issues/511 😅